marks-eduhub/skiller-frontend
# Feat: enable re-routing

# Description

This PR enables rerouting if a user is logged in  to the dashboard and if they are not to the auth screen

## Type of change


## Trello Ticket ID

https://trello.com/c/8tyvKs0J

## How Can This Been Tested?

Run the Branch, any credentials should login to the app and you can log out while you are inside (dashboard).
If you are  not logged in, it will not allow you to access the app


## Checklist:


# Screenshots

No
